app/src/main/java/com/devshish/internship/presentation/service/player/MediaBrowserService.kt

```diff
@@ -44,6 +59,20 @@ class MediaBrowserService : MediaBrowserServiceCompat() {
         }
     }
 
+    private fun createNotificationChannel() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            val name = "channel_name"
+            val descriptionText = "channel_desc"
```

Comment:
1. Lets hide this in some separat e class
2. It will be cool to provide some normal name and description